editor/editor_about.cpp

```diff
@@ -141,7 +141,7 @@ EditorAbout::EditorAbout() {
 	version_btn = memnew(LinkButton);
 	String hash = String(VERSION_HASH);
 	if (hash.length() != 0) {
-		hash = "." + hash.left(9);
+		hash = " " + vformat("[%s]", hash.left(9));
```

Comment:
Pushed square brackets version, looks good to me. I could embed the space directly in `vformat`, but then I think it's better to be explicit here, so it doesn't seem like a typo.